pkg/virt-api/rest/subresource_test.go

```diff
@@ -99,6 +115,19 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 
 	app := SubresourceAPIApp{}
 	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		kubeClient = fake.NewSimpleClientset()
+		virtClient = kubecli.NewMockKubevirtClient(ctrl)
+		vmClient = kubecli.NewMockVirtualMachineInterface(ctrl)
+		vmiClient = kubecli.NewMockVirtualMachineInstanceInterface(ctrl)
+		migrateClient = kubecli.NewMockVirtualMachineInstanceMigrationInterface(ctrl)
+
+		virtClient.EXPECT().CoreV1().Return(kubeClient.CoreV1()).AnyTimes()
+		virtClient.EXPECT().VirtualMachine(k8smetav1.NamespaceDefault).Return(vmClient).AnyTimes()
+		virtClient.EXPECT().VirtualMachine("").Return(vmClient).AnyTimes()
```

Comment:
Is this a leftover?
Referring to `virtClient.EXPECT().VirtualMachine("").Return(vmClient).AnyTimes()`.

Reply:
Thanks @iholder-redhat some tests (i.e. [this one](https://github.com/fossedihelm/kubevirt/blob/97ce2f4ee062f5cc211efbd129080b483cfa67a4/pkg/virt-api/rest/subresource_test.go#L301)) check the failure if the namespace is not passed down, for that reason I added this line. If there is another way to manage it, please give me an advice :slightly_smiling_face: